Sources/App/Controllers/PackageController+routes.swift

```diff
@@ -55,6 +55,115 @@ struct PackageController {
         }
     }
 
+    enum Fragment: String {
+        case documentation
+        case css
+        case data
+        case js
+    }
+
+    func awsURL(owner: String, repository: String, reference: String, fragment: Fragment, path: String) throws -> URI {
+        guard let bucket = Current.awsDocsBucket() else {
+            throw AppError.envVariableNotSet("AWS_DOCS_BUCKET")
+        }
+
+        return URI(string: "http://\(bucket).s3.us-east-2.amazonaws.com/\(owner)/\(repository)/\(reference)/\(fragment)/\(path)")
```

Comment:
We should probably make the AWS region configurable at some point but I didn't want to add another config variable right now.